plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -833,6 +978,40 @@ public static Object of(
         }
     }
 
+    @Override
+    @Test
+    public void testShowCreateTable()
+    {
+        assertQueryFails("SHOW CREATE TABLE orders", ".* Table 'pinot.default.orders' does not exist$");
```

Comment:
Why did you change the table to orders? Please use region as the base test class.

Reply:
Ah, this one was a mistake.